cmd/search/main.go

```diff
@@ -285,9 +329,9 @@ func (o *options) MetadataFor(path string) (Result, error) {
 			}
 			if len(comments.Info.Summary) > 0 {
 				if len(comments.Info.Status) > 0 {
-					result.Name = fmt.Sprintf("Bug %d: %s %s", id, comments.Info.Summary, comments.Info.Status)
+					result.Name = fmt.Sprintf("Issue %d: %s %s", id, comments.Info.Summary, comments.Info.Status)
 				} else {
-					result.Name = fmt.Sprintf("Bug %d: %s", id, comments.Info.Summary)
+					result.Name = fmt.Sprintf("Issue %d: %s", id, comments.Info.Summary)
```

Comment:
Same here...
This seems suspect...it's inside a `bugs/` block of code?  Why switch it to `Issue`?

Reply:
Most likely some IDE refactoring went wrong. Fixed.